### PR TITLE
Fix param set issue

### PR DESCRIPTION
**Pull request by julianoes**
This fixes an issue reported by @douglaswsilva.

As part of this pull request I've also updated to the latest E90 camera definition file.

@Stifael or @potaito maybe you can send me the latest E50, ET, etc. camera definition files, so I can add them here.

### CODE REVIEW

File: core/mavlink_parameters.cpp [hunk 1]
```diff
@@ -264,7 +264,7 @@ void MAVLinkParameters::process_param_value(const mavlink_message_t &message)
         return;
     }
 
-    if (strncmp(work->param_name.c_str(), param_value.param_id, PARAM_ID_LEN) != 0) {
+    if (work->param_name.compare(param_value.param_id) != 0) {
```
Comment (JonasVautherin) [inline core/mavlink_parameters.cpp]: I'm not sure I get the difference. What was the issue with the previous one?

Comment (julianoes) [reply core/mavlink_parameters.cpp]: Have you read the commit description?

Comment (JonasVautherin) [reply core/mavlink_parameters.cpp]: I didn't :sweat_smile:. I should get used to that, but I'm unfortunately not used to finding useful information in commit descriptions. Will work on it, thanks!

Comment (julianoes) [reply core/mavlink_parameters.cpp]: Hehe, I try to write make them explanatory.

**Comment by potaito** [comment]
@julianoes That would be nice, thanks. But where do I find these definition files and why would they change?

**Comment by julianoes** [comment]
They change as cameras get new features or fixes.

**Comment by julianoes** [comment]
Ideally the SDK downloads the files via http but that's currently broken through MFi on ST10C.

**Pull request merged**